projects/addon-charts/components/line-chart/line-chart.component.ts

```diff
@@ -99,6 +109,21 @@ export class TuiLineChartComponent {
 
     value: readonly TuiPoint[] = [];
 
+    hovered$ = this._hovered$.pipe(distinctUntilChanged(), tuiZoneOptimized(this.ngZone));
+
+    hoveredOverAxes$ = this.hints$.pipe(map(values => !!values.length)).pipe(
```

Comment:
`hoveredOverAxes$` exists only to patch `hovered$` with `NaN`-value (at a certain time), am i right ?

Could we integrate this logic inside `hovered$`'s pipe ?

Reply:
Possibly, it can help to drop extra subscription 🤔